OpenEBS import blog articles (#147)
openebs/website: website/src/blogs/running-through-the-mud-looking-skywards.md

---
title: Running through the mud looking skywards
author: Amit Kumar Das
author_info: Engineer the DAO
date: 2017-02-11
tags: Education, Industry, Storage, Openebs
excerpt: I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical.
---

> I knew, we would be running through the mud looking skywards for those brightest stars. We were not at all disappointed with our first attempt at *Kuppam Engineering College*, India.

Reviewer: ```suggestion
I knew we would be running through the mud looking skywards for those brightest stars. We were not at all disappointed with our first attempt at *Kuppam Engineering College*, India.
```

> I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical. We call ourselves as the industry’s core and form the industry’s grey cells. What we need to do is make our problems open to the public. These problems becomes the new practice that needs to be picked up by our *alma mater*. Once selected, it will take no time for these problems to go viral among the students of these institutions. Let these young minds chart their own course; love, enjoy & live every moment when they traverse through this meandering amazon of practical problems. This becomes the fodder for the future theory allowing the current one to rest in peace.

Reviewer: ```suggestion
I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical. We call ourselves the industry’s core and form the industry’s grey cells. What we need to do is make our problems open to the public. These problems becomes the new practice that needs to be picked up by our *alma mater*. Once selected, it will take no time for these problems to go viral among the students of these institutions. Let these young minds chart their own course; love, enjoy & live every moment when they traverse through this meandering amazon of practical problems. This becomes the fodder for the future theory allowing the current one to rest in peace.
```

> Above sounds like a grand planning that resembles more like a trickle down effect. Well the answer is a big ‘No’. Team at [**OpenEBS**](http://openebs.io/) believes there are many faster ways to achieve above in a time bound manner. Hence the formation of various internship programs where [**OpenEBS**](http://openebs.io/) approaches the Universities to get the latter on same plane that the former is passionate about.

Reviewer: ```suggestion
Above sounds like a grand planning that resembles more like a trickle down effect. Well, the answer is a big ‘No’. Team at [**OpenEBS**](http://openebs.io/) believes there are many faster ways to achieve above in a time bound manner. Hence the formation of various internship programs where [**OpenEBS**](http://openebs.io/) approaches the Universities to get the latter on same plane that the former is passionate about.
```

> **At Kuppam, we could feel the freshness in the air, the disruptions that yearns to come out of the closet. We are all geared to lighten up this brilliance which is currently lying dishevelled.**

Reviewer: ```suggestion
**At Kuppam, we could feel the freshness in the air, the disruptions that yearn to come out of the closet. We are all geared to lighten up this brilliance that is currently lying dishevelled.**
```

> On a closing note, I challenge the myriad number of B.E projects & PhD programs that are purposeless versus the likes of problems that the world wants us to solve. Join me in my effort to solve the problems that matters.

Reviewer: ```suggestion
On a closing note, I challenge the myriad number of B.E projects & Ph.D. programs that are purposeless versus the likes of problems that the world wants us to solve. Join me in my effort to solve the problems that matter.
```
